x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlNativeAutodetectIntegTestCase.java

```diff
@@ -290,17 +290,25 @@ protected String forecast(String jobId, TimeValue duration, TimeValue expiresIn,
     }
 
     protected void waitForecastToFinish(String jobId, String forecastId) throws Exception {
-        waitForecastStatus(jobId, forecastId, ForecastRequestStats.ForecastRequestStatus.FINISHED);
+        // Forecasts can take an eternity to complete in the FIPS JVM
+        waitForecastStatus(inFipsJvm() ? 300 : 60, jobId, forecastId, ForecastRequestStats.ForecastRequestStatus.FINISHED);
     }
 
     protected void waitForecastStatus(String jobId,
                                       String forecastId,
                                       ForecastRequestStats.ForecastRequestStatus... status) throws Exception {
+        waitForecastStatus(30, jobId, forecastId, status);
```

Comment:
Why is the default here (`30`) different than it is above (`60`)?

Reply:
The wait was 30 everywhere prior to #64471 (which relates to the same test failure in the forecast overflow to disk tests).  It was increased to 60 in that PR, but the only place a higher value was ever required was when waiting for a forecast to finish, not for waits on other statuses.  So now there is the opportunity to have a different length of wait for finish I put the length of wait for other statuses back to the original level.

Reply:
Great, thanks for explanation!